pkg/ccl/importccl/import_stmt.go

```diff
@@ -52,10 +52,11 @@ const (
 	csvNullIf    = "nullif"
 	csvSkip      = "skip"
 
-	mysqlOutfileRowSep   = "rows_terminated_by"
-	mysqlOutfileFieldSep = "fields_terminated_by"
-	mysqlOutfileEnclose  = "fields_enclosed_by"
-	mysqlOutfileEscape   = "fields_escaped_by"
+	mysqlOutfileRowSep      = "rows_terminated_by"
+	mysqlOutfileFieldSep    = "fields_terminated_by"
+	mysqlOutfileEnclose     = "fields_enclosed_by"
+	mysqlOutfileEscape      = "fields_escaped_by"
+	mysqlOutfileIgnoreLines = "ignore_lines"
```

Comment:
can we not use the existing `skip` option?

Reply:
done